feat(nova): 🌸 display styled zone labels in grid layout on index view OC:5940

carlacupani opened the pull request:
- Render the user's zones as styled tags within a responsive grid layout on the Nova index view.
- Applied custom HTML and inline CSS for improved readability, responsiveness, and aesthetics.
- Limited to visible zones where the user is an admin.

carlacupani committed:
feat(nova): 🌸 display styled zone labels in grid layout on index view OC:5940

- Render the user's zones as styled tags within a responsive grid layout on the Nova index view.
- Applied custom HTML and inline CSS for improved readability, responsiveness, and aesthetics.
- Limited to visible zones where the user is an admin.

peppedeka commented on app/Nova/PushNotification.php:
```diff
@@ -70,8 +72,47 @@ public function fields(NovaRequest $request)
+                    ->pluck('label')
+                    ->toArray();
+
+                return '<div style="
```
utilizzare here doc https://www.mrw.it/lez/php-sintassi-heredoc-nowdoc/

carlacupani committed:
[refactor] ♻️ Use HEREDOC for zone label rendering OC: 5940

- Rewrote HTML generation for zone labels using HEREDOC syntax for better readability and maintainability.
- Replaced string concatenation with HEREDOC
- Removed intermediate variables
- Preserved original layout and functionality
- Cleaned up indentation and style
